main.py

```diff
@@ -20,26 +18,26 @@
 import numpy as np
 import tensorflow as tf
 import config
+import tf_config
 import math
-# for saving to xml files
 from pascal_voc_writer import Writer
 
-
-def get_session():
-    config = tf.ConfigProto()
-    config.gpu_options.allow_growth = True
-    return tf.Session(config=config)
-
-
-keras.backend.tensorflow_backend.set_session(get_session())
-
-model_path = os.path.join('.', 'snapshots', 'resnet50_coco_best_v2.1.0.h5')
-
-model = models.load_model(model_path, backbone_name='resnet50')
+# make sure the file is inside semi-auto-image-annotation-tool-master
+import pathlib
+cur_path = pathlib.Path(__file__).parent.absolute()
+# cur_path = r"semi-auto-image-annotation-tool-master"
+sys.path.append(cur_path)
+os.chdir(cur_path)
 
 
 class MainGUI:
     def __init__(self, master):
+
+        # to choose between keras or tensorflow models
+        self.kera = 1  # default
+        self.tensorf = 0
```

Comment:
You might want to use PEP8 standards here.

`self.keras_` and `self.tensorflow_` can be good candidates for the variable names.